_data/projects.yml

```diff
@@ -522,14 +522,16 @@
     - Scala
 
 - title: "SETI@IBMCloud: SETI data, publicly available, from IBM"
+  features:
+    - seti
   headline: "Raw data from the SETI Institute for you to crunch in the search for extraterrestrial life."
   links: 
     GitHub: "https://github.com/ibm-watson-data-lab/seti_at_ibm"
     Medium: "https://medium.com/ibm-watson-data-lab/simulating-e-t-e34f4fa7a4f0"
```

Comment:
Hmm... this Medium article isn't technically part of SETI@IBMCloud. It's part of the ML4SETI code challenge. Those are two separate things.

Reply:
I see your point, but I think one has to be a subset of the other. So if you want that thing to be ml4seti, then that's fine. We can change the name.